db.json

```diff
@@ -3727,7 +3814,7 @@
   },
   "application/vnd.lotus-1-2-3": {
     "source": "iana",
-    "extensions": ["123"]
+    "extensions": ["123","wk1","wk3","wk4","wks"]
```

Comment:
.wks not mentioned on https://www.iana.org/assignments/media-types/application/vnd.lotus-1-2-3 , but it is an early version of the file format.